CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 # Changelog
 
+## v1.35.1 (2024-03-18)
+
+    [relase-1.35] CVE-2024-1753 container escape fix
```

Comment:
@TomSweeneyRedHat Just to give you the choice: if you want to respin with this typo fixed (also in the commit message where it came from), please merge afterwards without another review.

Alternatively, feel free to drop the hold and let this through.

Reply:
@mtrmac I completely missed that.  Thanks, fixed and repushed.